packages/devtools_app/lib/src/tree.dart

```diff
@@ -158,8 +187,16 @@ class _TreeViewItemState<T extends TreeNode<T>> extends State<TreeViewItem<T>>
     return dataObject.level * defaultSpacing;
   }
 
-  void _onPressed() {
-    widget.onItemPressed(widget.data);
+  void _onExpanded() {
+    if (widget.onItemExpanded == null) {
+      return;
+    }
+    widget.onItemExpanded(widget.data);
+    setExpanded(widget.data.isExpanded);
+  }
+
+  void _onSelected() {
+    widget.onItemSelected(widget.data);
```

Comment:
should we be checking that onItemSelected is not null here?

Reply:
Yup. Will add a null-aware operator.